### PR TITLE
CLOUD-1529 SCRIPT_DEBUG for dynamic resources

### PR DESCRIPTION
**Pull request by bdecoste**

### CODE REVIEW

File: dynamic-resources/dynamic_resources.sh [hunk 1]
```diff
@@ -1,5 +1,10 @@
 #!/bin/sh
 
+if [ "${SCRIPT_DEBUG}" = "true" ] ; then
```
Comment (bdecoste) [inline dynamic-resources/dynamic_resources.sh]: Is that a problem? I think that's a benefit.

Comment (rcernich) [inline dynamic-resources/dynamic_resources.sh]: Let me rephrase, what are the side effects, if any, of this being incorporated into the eap (and layered products) launch scripts?  This changes seems innocuous, but it affects more than just our config scripts.
